.github/workflows/pre-release.yml

```diff
@@ -70,6 +70,92 @@ jobs:
           track: alpha
 
+  deploy-app-store:
+    runs-on: macos-latest
+    timeout-minutes: 120
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Check if version has been updated
+        id: version_check
+        uses: EndBug/version-check@v1
+        with:
+          diff-search: true
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Import the Code-Signing PKCS12 Certificate
+        if: steps.version_check.outputs.changed == 'true'
+        uses: apple-actions/import-codesign-certs@v1
+        with:
+          p12-file-base64: ${{ secrets.CERTIFICATES_P12 }}
+          p12-password: ${{ secrets.CERTIFICATES_P12_PASSWORD }}
+      - uses: Apple-Actions/download-provisioning-profiles@v1
+        with:
+          bundle-id: io.numbersprotocol.capturelite
+          profile-type: 'IOS_APP_STORE'
+          issuer-id: ${{ secrets.APPSTORE_ISSUER_ID }}
+          api-key-id: ${{ secrets.APPSTORE_KEY_ID }}
+          api-private-key: ${{ secrets.APPSTORE_PRIVATE_KEY }}
+
+      - name: Get current time
+        uses: 1466587594/get-current-time@v2
+        id: current-time
+        with:
+          format: YYYYMMDDHHmmss
+
+      - name: Build Ionic
+        if: steps.version_check.outputs.changed == 'true'
+        env:
+          NUMBERS_STORAGE_BASE_URL: ${{ secrets.NUMBERS_STORAGE_BASE_URL }}
+        run: |
+          npm install -g @angular/cli
+          npm install -g @ionic/cli
+          npm install
+          npm run build
+
+      - name: Sync Capacitor web assets
+        if: steps.version_check.outputs.changed == 'true'
+        run: |
+          npx cap sync ios
+
+      - name: Build Xcode Archive
+        if: steps.version_check.outputs.changed == 'true'
+        run: |
+          xcodebuild archive -workspace ios/App/App.xcworkspace -scheme App -sdk iphoneos  -configuration Release -archivePath build/App.xcarchive -showBuildTimingSummary  MARKETING_VERSION=${{ steps.version_check.outputs.version }}  CURRENT_PROJECT_VERSION=${{ steps.current-time.outputs.formattedTime }}  -allowProvisioningUpdates archive
+
+      - name: Export IPA from Xcode Archive
+        if: steps.version_check.outputs.changed == 'true'
+        run: |
+          xcodebuild -exportArchive -archivePath build/App.xcarchive -exportPath build -exportOptionsPlist ios/App/ExportOptions.plist -allowProvisioningUpdates
+
+      - name: Upload IPA to App Store (Testflight)
+        uses: Apple-Actions/upload-testflight-build@master
+        if: steps.version_check.outputs.changed == 'true'
+        with:
+          app-path: build/App.ipa
+          issuer-id: ${{ secrets.APPSTORE_ISSUER_ID }}
+          api-key-id: ${{ secrets.APPSTORE_KEY_ID }}
+          api-private-key: ${{ secrets.APPSTORE_PRIVATE_KEY }}
+
+      - name: Upload IPA to artifacts
+        if: steps.version_check.outputs.changed == 'true'
+        uses: actions/upload-artifact@v1
```

Comment:
I think we should not upload large files, such as IPA or Xcode archive, to artifacts due to the storage space limitation. We could upload the files to Google drive if needed. The related secret keys and configuration has been already set. You can check the `upload-google-drive` for the deteiled action config.